### ISSUE TITLE
A missing && in a template leads to a cascade of errors in generated code

### ISSUE THREAD

**Issue by vsajip** [javacc21/javacc21#200]
https://github.com/javacc21/javacc21/blob/aa4db7e08448610489d7026773f12fcaed87a358/src/ftl/csharp/LookaheadRoutines.inc.ftl#L419
The line should be
```
${is}if (!ScanToEnd && _nonTerminalNesting <=1 && _lookaheadRoutineNesting <= 1) { 
```

**Comment by revusky** [comment]
Yeah, it stands to reason that it would be something like that.

**Comment by vsajip** [comment]
Should be fixed in 2aa61cb.
